src/constants/Forms.js

```diff
@@ -1,39 +1,50 @@
-export const FIELDS_SIGNUP = [
-  {
-    id: "first_name",
-    label: "Nombre",
-    type: "text",
+// import { rulesFields } from "./RulesFieldsUsers";
+const formFields = {
+  email: {
+    label: "Email",
+    name: "email",
+    type: "email",
   },
-  {
-    id: "last_name",
+
+  last_name: {
```

Comment:
```suggestion
  lastName: {
```
Use Camel case on variable names